### ISSUE TITLE
value error on export of model 

### ISSUE THREAD

**Issue by CrazyHairGuy** [SteamDatabase/ValveResourceFormat#309]
### Describe the bug
I get "value" error on exporting a model as glTF.

### Reference file in a Valve game
\Half-Life Alyx\game\hlvr\pak01_dir.vpk\models\creatures\headcrab_classic\headcrab_classic.vmdl_c

### Expected behavior
An exported glTF file

### Version info
VRF v0.1.6.1397, Windows 10 ver. 1909

### Additional context
Here's the console output:

``` ---> System.NotFiniteNumberException: value
   at SharpGLTF.Memory.FloatingAccessor.set_Item(Int32 rowIndex, Int32 subIndex, Single value)
   at SharpGLTF.Memory.EncodedArrayUtils._CopyTo[T](IEnumerable`1 src, IList`1 dst, Int32 dstOffset)
   at SharpGLTF.Schema2.AnimationSampler._CreateOutputAccessor(IReadOnlyList`1 output)
   at SharpGLTF.Schema2.AnimationSampler.SetKeys(IReadOnlyDictionary`2 keyframes)
   at ValveResourceFormat.IO.GltfModelExporter.<ExportToFile>g__AddMeshNode|14_0(String name, Mesh mesh, Skeleton skeleton, <>c__DisplayClass14_0& ) in C:\projects\valveresourceformat\ValveResourceFormat\IO\GltfModelExporter.cs:line 101
   at ValveResourceFormat.IO.GltfModelExporter.ExportToFile(String resourceName, String fileName, Model model) in C:\projects\valveresourceformat\ValveResourceFormat\IO\GltfModelExporter.cs:line 121
   at GUI.Types.Exporter.ExportFile.<>c__DisplayClass1_0.<Export>b__0(Object _, EventArgs __) in C:\projects\valveresourceformat\GUI\Types\Exporter\ExportFile.cs:line 88
   at GUI.Forms.GenericProgressForm.<OnShown>b__6_0() in C:\projects\valveresourceformat\GUI\Forms\GenericProgressForm.cs:line 37
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
   --- End of inner exception stack trace ---```

**Comment by xPaw** [comment]
![image](https://user-images.githubusercontent.com/613331/99804101-25af9800-2b43-11eb-90d6-04d7e218b7e8.png)

`W` in `FrameBone.Angle` is NaN for some reason, which causes this crash.

**Comment by xPaw** [comment]
This also happens in Dota 2 now with `:models\heroes\dawnbreaker\dawnbreaker.vmdl_c`.

**Comment by xPaw** [comment]
I think I fixed this with 403336c988fd4be85e748830c9e38e2256b0a6cc, can you give latest unstable build a go?

**Comment by CrazyHairGuy** [comment]
The headcrab model no longer gives an error on export, looks like it worked: [https://gfycat.com/ThirstyWelldocumentedFoxhound](url)
